main.js

```diff
@@ -0,0 +1,113 @@
+function Library(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
+class InterfaceUser {
+  static displayBooks() {
+    const books = [
+      {
+        title: 'Nuremberg Process',
+        author: 'Joe J. Heidecker',
+      },
+      {
+        title: 'The Science Fiction Hall of Fame',
+        author: 'Isaac Asimov',
+      },
+      {
+        title: 'The Martian',
+        author: 'Andy Weir',
+      },
+    ];
+    const validataLocalStorage = JSON.parse(localStorage.getItem('List'));
+    const parseObjt = validataLocalStorage;
+    if (parseObjt != null) {
+      for (let k = 0; k < parseObjt.length; k += 1) {
+        books.push(parseObjt[k]);
+      }
+    }
+    books.forEach((book) => { InterfaceUser.addBookToLibrary(book); });
+  }
+
+  static addBookToLibrary(book) {
+    const library = document.getElementById('book_list');
+    const row = document.createElement('section');
+
+    row.innerHTML = `
+      <div class='book'>
+        <div class="title">"${book.title}" ${'by'} ${book.author}</div>
+        <div><button href="#" class='delete'>Remove</button></div><hr>
+      </div>
+    `;
+
+    library.appendChild(row);
+  }
+
+  static deleteBook(element) {
+    if (element.classList.contains('delete')) {
+      element.parentElement.parentElement.remove();
+    }
+  }
```

Comment:
- [ ] Excellent job with the add book implementation. However, the "delete" book is not working properly. I noticed that all deleted books come back every time you reload the page, and that's because you don't update local storage when you delete a book. Please make sure your local storage is updated when a book is deleted from the list 😉